Run examples using Intellij Idea

Hello,

Would you be able to run examples under intellij?
I've tried to run an example on mac os using intellij idea and I've received next error:

<details><summary>click to expand</summary><p>

#### console output: 
```
Nov 09, 2017 8:53:54 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,main] has been blocked for 2549 ms, time limit is 2000
Nov 09, 2017 8:53:54 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,main] has been blocked for 2839 ms, time limit is 2000
Nov 09, 2017 8:53:55 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,main] has been blocked for 3552 ms, time limit is 2000
Nov 09, 2017 8:53:55 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,main] has been blocked for 3841 ms, time limit is 2000
Nov 09, 2017 8:53:56 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,main] has been blocked for 4557 ms, time limit is 2000
Nov 09, 2017 8:53:56 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,main] has been blocked for 4846 ms, time limit is 2000
.Compile error(s): [ERROR]Error: Could not find or load main class Support.JetBrains.Toolbox.apps.IDEA-U.ch-0.172.4343.14.IntelliJ

Nov 09, 2017 8:53:57 PM live.connector.vertxui.samples.server.AllExamplesServer lambda$start$1
SEVERE: Startup error
java.net.SocketException: Permission denied
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:223)
	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:128)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:554)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1258)
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:502)
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:487)
	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:980)
	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:250)
	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:365)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasksFrom(SingleThreadEventExecutor.java:379)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:354)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:436)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:745)
```
</p></details>

I will try to describe all steps I did:
Prerequisites: mac os 10.13, java 1.8.0_151.
1. Imorted 2 projects (as maven projects) vertxui-core and vertxui-examples
2. Tried to build vertxui-core with command mvn clean install. Output=>
`Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar (attach-javadocs) on project vertxui-core: MavenReportException: Error while creating archive:
[ERROR] Exit code: 1 - /vertxui/vertxui-core/src/main/java/live/connector/vertxui/server/FigWheelyServer.java:25: error: tag not allowed here: <dd>
[ERROR] * <dd>vertx.eventBus().publish(FigWheely.figNotify, "bla die bla");
[ERROR] ^
[ERROR] /vertxui/vertxui-core/src/main/java/live/connector/vertxui/client/FigWheelyClient.java:20: warning: no description for @param
[ERROR] * @param message
[ERROR] ^
[ERROR] 
[ERROR] Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/bin/javadoc @options @packages
[ERROR] `
3. Removed <dd> in FigWheelyServer line 25 and run build again. Output =>
`[INFO] --- maven-gpg-plugin:1.5:sign (sign-artifacts) @ vertxui-core ---
/bin/sh: gpg: command not found
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 5.219 s
[INFO] Finished at: 2017-11-14T11:38:59+01:00
[INFO] Final Memory: 38M/517M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.5:sign (sign-artifacts) on project vertxui-core: Exit code: 127 -> [Help 1]`
4. Disable plugin maven-gpg-plugin in vertxui-core pom. Output =>
`[INFO] BUILD SUCCESS`
5. Build vertxui-examples with command mvn clean package. Output=>
`[INFO] Building jar: /vertxui/vertxui-examples/target/vertxui-samples-1.01.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS`
6. Run mvn exec:java -Dexec.mainClass="live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap". Output=>
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for live.connector:vertxui-samples:jar:1.01
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: junit:junit:jar -> duplicate declaration of version 4.12 @ line 55, column 15
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building live.connector:vertxui-samples 1.01
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- exec-maven-plugin:1.6.0:java (default-cli) @ vertxui-samples ---
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Nov 14, 2017 11:55:29 AM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 2704 ms, time limit is 2000
Nov 14, 2017 11:55:29 AM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 2922 ms, time limit is 2000
Nov 14, 2017 11:55:30 AM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 3708 ms, time limit is 2000
Nov 14, 2017 11:55:30 AM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 3927 ms, time limit is 2000
Nov 14, 2017 11:55:31 AM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 4712 ms, time limit is 2000
Nov 14, 2017 11:55:31 AM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 4930 ms, time limit is 2000
**.Compile error(s): [ERROR]Error: Could not find or load main class com.google.gwt.dev.Compiler**

Nov 14, 2017 11:55:32 AM live.connector.vertxui.samples.server.AllExamplesServer lambda$start$1
INFO: Initialised:
        http://localhost:8081/rest/totals
        http://localhost:8081/rest/bills
        http://localhost:8081/rest/grocery
        http://localhost:8081/figwheely.js
        http://localhost:8081/
`
I run all commands from console to exclude dependency on IDE.

Many thanks for your update.
I don't think that problem with IDE. All next steps were done from console and I cloned your repository again from scratch. 
1. Tried to build vertxui-core and received next error:
#### console output: 

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.5:sign (sign-artifacts) on project vertxui-core: Exit code: 127 -> [Help 1]
```
 So I just disabled this plugin:
![image](https://user-images.githubusercontent.com/17416455/32800021-9128d742-c979-11e7-9636-0663786d92db.png)
And was able to build vertxui-core.

2. Then I tried to build vertxui-examples and got this:
<details><summary>click to expand</summary><p>

#### console output: 

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project vertxui-samples: Compilation failure: Compilation failure:
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithDom/AnotherTest.java:[12,17] package org.junit does not exist
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithDom/TestjUnitWithDom.java:[10,17] package org.junit does not exist
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[3,24] package org.junit does not exist
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[3,1] static import only from classes and interfaces
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[4,24] package org.junit does not exist
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[4,1] static import only from classes and interfaces
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[8,17] package org.junit does not exist
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithDom/AnotherTest.java:[25,10] cannot find symbol
[ERROR]   symbol:   class Test
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithDom.AnotherTest
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithDom/TestjUnitWithDom.java:[51,10] cannot find symbol
[ERROR]   symbol:   class Test
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithDom.TestjUnitWithDom
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[33,10] cannot find symbol
[ERROR]   symbol:   class Test
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithoutDom.TestjUnitWithoutDom
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[53,10] cannot find symbol
[ERROR]   symbol:   class Test
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithoutDom.TestjUnitWithoutDom
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[75,10] cannot find symbol
[ERROR]   symbol:   class Test
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithoutDom.TestjUnitWithoutDom
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[49,17] cannot find symbol
[ERROR]   symbol:   method assertTrue(boolean)
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithoutDom.TestjUnitWithoutDom
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[50,17] cannot find symbol
[ERROR]   symbol:   method assertTrue(boolean)
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithoutDom.TestjUnitWithoutDom
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[71,17] cannot find symbol
[ERROR]   symbol:   method assertEquals(int,int)
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithoutDom.TestjUnitWithoutDom
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[72,17] cannot find symbol
[ERROR]   symbol:   method assertTrue(boolean)
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithoutDom.TestjUnitWithoutDom
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[81,17] cannot find symbol
[ERROR]   symbol:   method assertEquals(int,int)
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithoutDom.TestjUnitWithoutDom
[ERROR] /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/live/connector/vertxui/samples/client/testjUnitWithoutDom/TestjUnitWithoutDom.java:[82,17] cannot find symbol
[ERROR]   symbol:   method assertEquals(int,int)
[ERROR]   location: class live.connector.vertxui.samples.client.testjUnitWithoutDom.TestjUnitWithoutDom
```
</p></details>

I removed scope test from pom:
![image](https://user-images.githubusercontent.com/17416455/32800144-db66076c-c979-11e7-99a0-107108f33b4b.png)

3. Everything looks good and now I can build both projects.
4. Next step is to run any example under directory  vertxui-examples: 
```
mvn exec:java -Dexec.mainClass="live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap"
```
<details><summary>click to expand</summary>

#### console output: 

```
INFO] Scanning for projects...
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building live.connector:vertxui-samples 1.01
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- exec-maven-plugin:1.6.0:java (default-cli) @ vertxui-samples ---
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Classpath while running does not contain GWT for translating java to javascript.System property java.class.path is: "/usr/local/Cellar/maven/3.5.0/libexec/boot/plexus-classworlds-2.5.2.jar:src/main/java"Nov 14, 2017 9:29:23 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 2708 ms, time limit is 2000
Nov 14, 2017 9:29:23 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 2924 ms, time limit is 2000
Nov 14, 2017 9:29:24 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 3711 ms, time limit is 2000
Nov 14, 2017 9:29:24 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 3927 ms, time limit is 2000
Nov 14, 2017 9:29:25 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 4715 ms, time limit is 2000
Nov 14, 2017 9:29:25 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,live.connector.vertxui.samples.server.mvcBootstrap.ServerBootstrap] has been blocked for 4931 ms, time limit is 2000
Nov 14, 2017 9:29:26 PM live.connector.vertxui.samples.server.AllExamplesServer lambda$start$1
INFO: Initialised:
        http://localhost:8088/rest/totals
        http://localhost:8088/rest/bills
        http://localhost:8088/rest/grocery
        http://localhost:8088/figwheely.js
        http://localhost:8088/

```
</details>

So no errors but I can't see any UI components:

![image](https://user-images.githubusercontent.com/17416455/32803641-790409e2-c984-11e7-9b60-67c42d31fda3.png)

Sorry my fault, but if I set configuration like it was before 
![image](https://user-images.githubusercontent.com/17416455/33395008-7578f6fe-d544-11e7-9561-0a071ed5c488.png)
and output is: 
```
objc[12106]: Class JavaLaunchHelper is implemented in both /Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/bin/java (0x1064a64c0) and /Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/libinstrument.dylib (0x1065324e0). One of the two will be used. Which one is undefined.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Nov 29, 2017 8:30:33 PM live.connector.vertxui.server.VertxUI translate
INFO: Translating with targetfolder=/Users/tol/work/java/vertxui/vertxui-examples/build/development
Nov 29, 2017 8:30:33 PM live.connector.vertxui.server.VertxUI translate
INFO: 	sourceFolder/Users/tol/work/java/vertxui/vertxui-examples/src/main/java
Nov 29, 2017 8:30:33 PM live.connector.vertxui.server.VertxUI translate
INFO: 	working folder: /Users/tol/work/java/vertxui/vertxui-examples/.
Nov 29, 2017 8:30:33 PM live.connector.vertxui.server.VertxUI translate
INFO: Writing temporary index.html to: /Users/tol/work/java/vertxui/vertxui-examples/build/development/index.html
Nov 29, 2017 8:30:33 PM live.connector.vertxui.server.VertxUI translate
INFO: Writing gwt.xml to: /Users/tol/work/java/vertxui/vertxui-examples/src/main/java/gwtTemp.gwt.xml
Nov 29, 2017 8:30:33 PM live.connector.vertxui.server.VertxUI translate
INFO: Classpath = "/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/charsets.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/deploy.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/cldrdata.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/dnsns.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/jaccess.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/jfxrt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/localedata.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/nashorn.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/sunec.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/sunjce_provider.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/sunpkcs11.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/zipfs.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/javaws.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/jce.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/jfr.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/jfxswt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/jsse.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/management-agent.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/plugin.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/resources.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/rt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/ant-javafx.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/dt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/javafx-mx.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/jconsole.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/packager.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/sa-jdi.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/tools.jar:/Users/tol/work/java/vertxui/vertxui-examples/target/classes/:/Users/tol/work/java/vertxui/vertxui-core/target/classes/:/Users/tol/.m2/repository/io/vertx/vertx-web/3.4.1/vertx-web-3.4.1.jar:/Users/tol/.m2/repository/io/vertx/vertx-auth-common/3.4.1/vertx-auth-common-3.4.1.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-dev/2.8.0/gwt-dev-2.8.0.jar:/Users/tol/.m2/repository/com/google/code/gson/gson/2.6.2/gson-2.6.2.jar:/Users/tol/.m2/repository/org/ow2/asm/asm/5.0.3/asm-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-util/5.0.3/asm-util-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-tree/5.0.3/asm-tree-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-commons/5.0.3/asm-commons-5.0.3.jar:/Users/tol/.m2/repository/colt/colt/1.2.0/colt-1.2.0.jar:/Users/tol/.m2/repository/ant/ant/1.6.5/ant-1.6.5.jar:/Users/tol/.m2/repository/commons-collections/commons-collections/3.2.2/commons-collections-3.2.2.jar:/Users/tol/.m2/repository/commons-io/commons-io/2.4/commons-io-2.4.jar:/Users/tol/.m2/repository/com/ibm/icu/icu4j/50.1.1/icu4j-50.1.1.jar:/Users/tol/.m2/repository/tapestry/tapestry/4.0.2/tapestry-4.0.2.jar:/Users/tol/.m2/repository/net/sourceforge/htmlunit/htmlunit/2.19/htmlunit-2.19.jar:/Users/tol/.m2/repository/xalan/xalan/2.7.2/xalan-2.7.2.jar:/Users/tol/.m2/repository/xalan/serializer/2.7.2/serializer-2.7.2.jar:/Users/tol/.m2/repository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpclient/4.5.1/httpclient-4.5.1.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpcore/4.4.3/httpcore-4.4.3.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpmime/4.5.1/httpmime-4.5.1.jar:/Users/tol/.m2/repository/commons-codec/commons-codec/1.10/commons-codec-1.10.jar:/Users/tol/.m2/repository/net/sourceforge/htmlunit/htmlunit-core-js/2.17/htmlunit-core-js-2.17.jar:/Users/tol/.m2/repository/xerces/xercesImpl/2.11.0/xercesImpl-2.11.0.jar:/Users/tol/.m2/repository/xml-apis/xml-apis/1.4.01/xml-apis-1.4.01.jar:/Users/tol/.m2/repository/net/sourceforge/nekohtml/nekohtml/1.9.22/nekohtml-1.9.22.jar:/Users/tol/.m2/repository/net/sourceforge/cssparser/cssparser/0.9.18/cssparser-0.9.18.jar:/Users/tol/.m2/repository/commons-logging/commons-logging/1.2/commons-logging-1.2.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-client/9.2.13.v20150730/websocket-client-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-common/9.2.13.v20150730/websocket-common-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-api/9.2.13.v20150730/websocket-api-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-webapp/9.2.14.v20151106/jetty-webapp-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-xml/9.2.14.v20151106/jetty-xml-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-servlet/9.2.14.v20151106/jetty-servlet-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-security/9.2.14.v20151106/jetty-security-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-servlets/9.2.14.v20151106/jetty-servlets-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-continuation/9.2.14.v20151106/jetty-continuation-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-http/9.2.14.v20151106/jetty-http-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-util/9.2.14.v20151106/jetty-util-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-io/9.2.14.v20151106/jetty-io-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-annotations/9.2.14.v20151106/jetty-annotations-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-plus/9.2.14.v20151106/jetty-plus-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-jndi/9.2.14.v20151106/jetty-jndi-9.2.14.v20151106.jar:/Users/tol/.m2/repository/javax/annotation/javax.annotation-api/1.2/javax.annotation-api-1.2.jar:/Users/tol/.m2/repository/org/eclipse/jetty/apache-jsp/9.2.14.v20151106/apache-jsp-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-server/9.2.14.v20151106/jetty-server-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/toolchain/jetty-schemas/3.1.M0/jetty-schemas-3.1.M0.jar:/Users/tol/.m2/repository/javax/servlet/javax.servlet-api/3.1.0/javax.servlet-api-3.1.0.jar:/Users/tol/.m2/repository/org/mortbay/jasper/apache-jsp/8.0.9.M3/apache-jsp-8.0.9.M3.jar:/Users/tol/.m2/repository/org/mortbay/jasper/apache-el/8.0.9.M3/apache-el-8.0.9.M3.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-elemental/2.8.0/gwt-elemental-2.8.0.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-user/2.8.0/gwt-user-2.8.0.jar:/Users/tol/.m2/repository/com/google/jsinterop/jsinterop-annotations/1.0.1/jsinterop-annotations-1.0.1.jar:/Users/tol/.m2/repository/com/google/jsinterop/jsinterop-annotations/1.0.1/jsinterop-annotations-1.0.1-sources.jar:/Users/tol/.m2/repository/javax/validation/validation-api/1.0.0.GA/validation-api-1.0.0.GA.jar:/Users/tol/.m2/repository/javax/validation/validation-api/1.0.0.GA/validation-api-1.0.0.GA-sources.jar:/Users/tol/.m2/repository/org/w3c/css/sac/1.3/sac-1.3.jar:/Users/tol/.m2/repository/com/github/nmorel/gwtjackson/gwt-jackson/0.14.2/gwt-jackson-0.14.2.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.8.4/jackson-annotations-2.8.4.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.8.4/jackson-annotations-2.8.4-sources.jar:/Users/tol/.m2/repository/com/squareup/javapoet/1.0.0/javapoet-1.0.0.jar:/Users/tol/.m2/repository/junit/junit/4.12/junit-4.12.jar:/Users/tol/.m2/repository/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar:/Users/tol/.m2/repository/com/machinepublishers/jbrowserdriver/0.17.5/jbrowserdriver-0.17.5.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpclient-cache/4.5.2/httpclient-cache-4.5.2.jar:/Users/tol/.m2/repository/org/slf4j/slf4j-api/1.7.2/slf4j-api-1.7.2.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-api/2.53.0/selenium-api-2.53.0.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-remote-driver/2.53.0/selenium-remote-driver-2.53.0.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-server/2.53.0/selenium-server-2.53.0.jar:/Users/tol/.m2/repository/com/google/guava/guava/19.0/guava-19.0.jar:/Users/tol/.m2/repository/org/zeroturnaround/zt-process/1.3/zt-process-1.3.jar:/Users/tol/.m2/repository/org/zeroturnaround/zt-exec/1.7/zt-exec-1.7.jar:/Users/tol/.m2/repository/com/sun/jna/jna/3.0.9/jna-3.0.9.jar:/Users/tol/.m2/repository/commons-lang/commons-lang/2.6/commons-lang-2.6.jar:/Users/tol/.m2/repository/io/github/lukehutch/fast-classpath-scanner/2.0.16/fast-classpath-scanner-2.0.16.jar:/Users/tol/.m2/repository/io/vertx/vertx-core/3.4.1/vertx-core-3.4.1.jar:/Users/tol/.m2/repository/io/netty/netty-common/4.1.8.Final/netty-common-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-buffer/4.1.8.Final/netty-buffer-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-transport/4.1.8.Final/netty-transport-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-handler/4.1.8.Final/netty-handler-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec/4.1.8.Final/netty-codec-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-handler-proxy/4.1.8.Final/netty-handler-proxy-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-socks/4.1.8.Final/netty-codec-socks-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-http/4.1.8.Final/netty-codec-http-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-http2/4.1.8.Final/netty-codec-http2-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-resolver/4.1.8.Final/netty-resolver-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-resolver-dns/4.1.8.Final/netty-resolver-dns-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-dns/4.1.8.Final/netty-codec-dns-4.1.8.Final.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.7.4/jackson-core-2.7.4.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.7.4/jackson-databind-2.7.4.jar:/Users/tol/Library/Application%20Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/172.4343.14/IntelliJ%20IDEA.app/Contents/lib/idea_rt.jar:/Users/tol/work/java/vertxui/vertxui-examples/src/main/java"
Nov 29, 2017 8:30:33 PM live.connector.vertxui.server.VertxUI translate
INFO: Starting GWT with commandline: java -cp "/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/charsets.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/deploy.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/cldrdata.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/dnsns.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/jaccess.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/jfxrt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/localedata.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/nashorn.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/sunec.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/sunjce_provider.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/sunpkcs11.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/ext/zipfs.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/javaws.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/jce.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/jfr.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/jfxswt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/jsse.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/management-agent.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/plugin.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/resources.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/jre/lib/rt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/ant-javafx.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/dt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/javafx-mx.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/jconsole.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/packager.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/sa-jdi.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home/lib/tools.jar:/Users/tol/work/java/vertxui/vertxui-examples/target/classes/:/Users/tol/work/java/vertxui/vertxui-core/target/classes/:/Users/tol/.m2/repository/io/vertx/vertx-web/3.4.1/vertx-web-3.4.1.jar:/Users/tol/.m2/repository/io/vertx/vertx-auth-common/3.4.1/vertx-auth-common-3.4.1.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-dev/2.8.0/gwt-dev-2.8.0.jar:/Users/tol/.m2/repository/com/google/code/gson/gson/2.6.2/gson-2.6.2.jar:/Users/tol/.m2/repository/org/ow2/asm/asm/5.0.3/asm-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-util/5.0.3/asm-util-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-tree/5.0.3/asm-tree-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-commons/5.0.3/asm-commons-5.0.3.jar:/Users/tol/.m2/repository/colt/colt/1.2.0/colt-1.2.0.jar:/Users/tol/.m2/repository/ant/ant/1.6.5/ant-1.6.5.jar:/Users/tol/.m2/repository/commons-collections/commons-collections/3.2.2/commons-collections-3.2.2.jar:/Users/tol/.m2/repository/commons-io/commons-io/2.4/commons-io-2.4.jar:/Users/tol/.m2/repository/com/ibm/icu/icu4j/50.1.1/icu4j-50.1.1.jar:/Users/tol/.m2/repository/tapestry/tapestry/4.0.2/tapestry-4.0.2.jar:/Users/tol/.m2/repository/net/sourceforge/htmlunit/htmlunit/2.19/htmlunit-2.19.jar:/Users/tol/.m2/repository/xalan/xalan/2.7.2/xalan-2.7.2.jar:/Users/tol/.m2/repository/xalan/serializer/2.7.2/serializer-2.7.2.jar:/Users/tol/.m2/repository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpclient/4.5.1/httpclient-4.5.1.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpcore/4.4.3/httpcore-4.4.3.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpmime/4.5.1/httpmime-4.5.1.jar:/Users/tol/.m2/repository/commons-codec/commons-codec/1.10/commons-codec-1.10.jar:/Users/tol/.m2/repository/net/sourceforge/htmlunit/htmlunit-core-js/2.17/htmlunit-core-js-2.17.jar:/Users/tol/.m2/repository/xerces/xercesImpl/2.11.0/xercesImpl-2.11.0.jar:/Users/tol/.m2/repository/xml-apis/xml-apis/1.4.01/xml-apis-1.4.01.jar:/Users/tol/.m2/repository/net/sourceforge/nekohtml/nekohtml/1.9.22/nekohtml-1.9.22.jar:/Users/tol/.m2/repository/net/sourceforge/cssparser/cssparser/0.9.18/cssparser-0.9.18.jar:/Users/tol/.m2/repository/commons-logging/commons-logging/1.2/commons-logging-1.2.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-client/9.2.13.v20150730/websocket-client-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-common/9.2.13.v20150730/websocket-common-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-api/9.2.13.v20150730/websocket-api-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-webapp/9.2.14.v20151106/jetty-webapp-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-xml/9.2.14.v20151106/jetty-xml-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-servlet/9.2.14.v20151106/jetty-servlet-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-security/9.2.14.v20151106/jetty-security-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-servlets/9.2.14.v20151106/jetty-servlets-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-continuation/9.2.14.v20151106/jetty-continuation-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-http/9.2.14.v20151106/jetty-http-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-util/9.2.14.v20151106/jetty-util-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-io/9.2.14.v20151106/jetty-io-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-annotations/9.2.14.v20151106/jetty-annotations-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-plus/9.2.14.v20151106/jetty-plus-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-jndi/9.2.14.v20151106/jetty-jndi-9.2.14.v20151106.jar:/Users/tol/.m2/repository/javax/annotation/javax.annotation-api/1.2/javax.annotation-api-1.2.jar:/Users/tol/.m2/repository/org/eclipse/jetty/apache-jsp/9.2.14.v20151106/apache-jsp-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-server/9.2.14.v20151106/jetty-server-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/toolchain/jetty-schemas/3.1.M0/jetty-schemas-3.1.M0.jar:/Users/tol/.m2/repository/javax/servlet/javax.servlet-api/3.1.0/javax.servlet-api-3.1.0.jar:/Users/tol/.m2/repository/org/mortbay/jasper/apache-jsp/8.0.9.M3/apache-jsp-8.0.9.M3.jar:/Users/tol/.m2/repository/org/mortbay/jasper/apache-el/8.0.9.M3/apache-el-8.0.9.M3.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-elemental/2.8.0/gwt-elemental-2.8.0.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-user/2.8.0/gwt-user-2.8.0.jar:/Users/tol/.m2/repository/com/google/jsinterop/jsinterop-annotations/1.0.1/jsinterop-annotations-1.0.1.jar:/Users/tol/.m2/repository/com/google/jsinterop/jsinterop-annotations/1.0.1/jsinterop-annotations-1.0.1-sources.jar:/Users/tol/.m2/repository/javax/validation/validation-api/1.0.0.GA/validation-api-1.0.0.GA.jar:/Users/tol/.m2/repository/javax/validation/validation-api/1.0.0.GA/validation-api-1.0.0.GA-sources.jar:/Users/tol/.m2/repository/org/w3c/css/sac/1.3/sac-1.3.jar:/Users/tol/.m2/repository/com/github/nmorel/gwtjackson/gwt-jackson/0.14.2/gwt-jackson-0.14.2.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.8.4/jackson-annotations-2.8.4.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.8.4/jackson-annotations-2.8.4-sources.jar:/Users/tol/.m2/repository/com/squareup/javapoet/1.0.0/javapoet-1.0.0.jar:/Users/tol/.m2/repository/junit/junit/4.12/junit-4.12.jar:/Users/tol/.m2/repository/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar:/Users/tol/.m2/repository/com/machinepublishers/jbrowserdriver/0.17.5/jbrowserdriver-0.17.5.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpclient-cache/4.5.2/httpclient-cache-4.5.2.jar:/Users/tol/.m2/repository/org/slf4j/slf4j-api/1.7.2/slf4j-api-1.7.2.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-api/2.53.0/selenium-api-2.53.0.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-remote-driver/2.53.0/selenium-remote-driver-2.53.0.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-server/2.53.0/selenium-server-2.53.0.jar:/Users/tol/.m2/repository/com/google/guava/guava/19.0/guava-19.0.jar:/Users/tol/.m2/repository/org/zeroturnaround/zt-process/1.3/zt-process-1.3.jar:/Users/tol/.m2/repository/org/zeroturnaround/zt-exec/1.7/zt-exec-1.7.jar:/Users/tol/.m2/repository/com/sun/jna/jna/3.0.9/jna-3.0.9.jar:/Users/tol/.m2/repository/commons-lang/commons-lang/2.6/commons-lang-2.6.jar:/Users/tol/.m2/repository/io/github/lukehutch/fast-classpath-scanner/2.0.16/fast-classpath-scanner-2.0.16.jar:/Users/tol/.m2/repository/io/vertx/vertx-core/3.4.1/vertx-core-3.4.1.jar:/Users/tol/.m2/repository/io/netty/netty-common/4.1.8.Final/netty-common-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-buffer/4.1.8.Final/netty-buffer-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-transport/4.1.8.Final/netty-transport-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-handler/4.1.8.Final/netty-handler-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec/4.1.8.Final/netty-codec-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-handler-proxy/4.1.8.Final/netty-handler-proxy-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-socks/4.1.8.Final/netty-codec-socks-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-http/4.1.8.Final/netty-codec-http-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-http2/4.1.8.Final/netty-codec-http2-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-resolver/4.1.8.Final/netty-resolver-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-resolver-dns/4.1.8.Final/netty-resolver-dns-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-dns/4.1.8.Final/netty-codec-dns-4.1.8.Final.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.7.4/jackson-core-2.7.4.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.7.4/jackson-databind-2.7.4.jar:/Users/tol/Library/Application%20Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/172.4343.14/IntelliJ%20IDEA.app/Contents/lib/idea_rt.jar:/Users/tol/work/java/vertxui/vertxui-examples/src/main/java" com.google.gwt.dev.Compiler -strict -XdisableUpdateCheck -war /Users/tol/work/java/vertxui/vertxui-examples/build/development -draftCompile -optimize 0 -style DETAILED gwtTemp
Nov 29, 2017 8:30:35 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,main] has been blocked for 2863 ms, time limit is 2000
Nov 29, 2017 8:30:35 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,main] has been blocked for 2601 ms, time limit is 2000
Nov 29, 2017 8:30:36 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,main] has been blocked for 3865 ms, time limit is 2000
Nov 29, 2017 8:30:36 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,main] has been blocked for 3603 ms, time limit is 2000
Nov 29, 2017 8:30:37 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-0,5,main] has been blocked for 4868 ms, time limit is 2000
Nov 29, 2017 8:30:37 PM io.vertx.core.impl.BlockedThreadChecker
WARNING: Thread Thread[vert.x-eventloop-thread-1,5,main] has been blocked for 4606 ms, time limit is 2000
Nov 29, 2017 8:30:38 PM live.connector.vertxui.server.VertxUI translateContinue
.INFO: Gwt says: Loading inherited module 'gwtTemp'
Nov 29, 2017 8:30:38 PM live.connector.vertxui.server.VertxUI translateContinue
INFO: Gwt says:    [ERROR] Unable to find 'gwtTemp.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?
.Nov 29, 2017 8:30:38 PM live.connector.vertxui.server.VertxUI writeHtml
INFO: Writing final index.html to: build/development/index.html
Compile error(s): Loading inherited module 'gwtTemp'
   [ERROR] Unable to find 'gwtTemp.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?

Nov 29, 2017 8:30:38 PM live.connector.vertxui.samples.server.AllExamplesServer lambda$start$1
INFO: Initialised:
	http://localhost:8088/rest/totals
	http://localhost:8088/rest/bills
	http://localhost:8088/rest/grocery
	http://localhost:8088/figwheely.js
	http://localhost:8088/

```

I understand but in this case probably there is a problem with initialisation ordering because I am recieving this error and I can see that gwtTemp.gwt.xml is created and removed but it has not included in classpath:

```
INFO: Gwt says: Loading inherited module 'gwtTemp'
Nov 30, 2017 9:28:27 PM live.connector.vertxui.server.VertxUI translateContinue
INFO: Gwt says:    [ERROR] Unable to find 'gwtTemp.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?
.Nov 30, 2017 9:28:27 PM live.connector.vertxui.server.VertxUI writeHtml
INFO: Writing final index.html to: build/development/index.html
Compile error(s): Loading inherited module 'gwtTemp'
   [ERROR] Unable to find 'gwtTemp.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?

Nov 30, 2017 9:28:27 PM live.connector.vertxui.samples.server.AllExamplesServer lambda$start$1
INFO: Initialised:
	http://localhost:8088/chatEventbus/
	http://localhost:8088/figwheely.js
	http://localhost:8088/
```

Hi,

I've just tried to run this on another computer but the same OS:

```
objc[20628]: Class JavaLaunchHelper is implemented in both /Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/bin/java (0x10406f4c0) and /Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/libinstrument.dylib (0x1041f04e0). One of the two will be used. Which one is undefined.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI with
INFO: source folder that will be used is src/main/java
Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI translate
INFO: Translating with targetfolder=/Volumes/HDD/work/vertxui/vertxui-examples/build/development
Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI translate
INFO: 	sourceFolder: /Volumes/HDD/work/vertxui/vertxui-examples/src/main/java
Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI translate
INFO: 	working folder: /Volumes/HDD/work/vertxui/vertxui-examples/.
Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI translate
INFO: Writing temporary index.html to: /Volumes/HDD/work/vertxui/vertxui-examples/build/development/index.html
Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI translate
INFO: Writing gwt.xml to: /Volumes/HDD/work/vertxui/vertxui-examples/src/main/java/gwtTemp.gwt.xml
Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI translate
INFO: Classpath = "/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/charsets.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/deploy.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/cldrdata.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/dnsns.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/jaccess.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/jfxrt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/localedata.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/nashorn.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/sunec.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/sunjce_provider.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/sunpkcs11.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/zipfs.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/javaws.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/jce.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/jfr.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/jfxswt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/jsse.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/management-agent.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/plugin.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/resources.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/rt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/ant-javafx.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/dt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/javafx-mx.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/jconsole.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/packager.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/sa-jdi.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/tools.jar:/Volumes/HDD/work/vertxui/vertxui-examples/target/classes/:/Volumes/HDD/work/vertxui/vertxui-core/target/classes/:/Users/tol/.m2/repository/io/vertx/vertx-web/3.4.1/vertx-web-3.4.1.jar:/Users/tol/.m2/repository/io/vertx/vertx-auth-common/3.4.1/vertx-auth-common-3.4.1.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-dev/2.8.0/gwt-dev-2.8.0.jar:/Users/tol/.m2/repository/com/google/code/gson/gson/2.6.2/gson-2.6.2.jar:/Users/tol/.m2/repository/org/ow2/asm/asm/5.0.3/asm-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-util/5.0.3/asm-util-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-tree/5.0.3/asm-tree-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-commons/5.0.3/asm-commons-5.0.3.jar:/Users/tol/.m2/repository/colt/colt/1.2.0/colt-1.2.0.jar:/Users/tol/.m2/repository/ant/ant/1.6.5/ant-1.6.5.jar:/Users/tol/.m2/repository/commons-collections/commons-collections/3.2.2/commons-collections-3.2.2.jar:/Users/tol/.m2/repository/commons-io/commons-io/2.4/commons-io-2.4.jar:/Users/tol/.m2/repository/com/ibm/icu/icu4j/50.1.1/icu4j-50.1.1.jar:/Users/tol/.m2/repository/tapestry/tapestry/4.0.2/tapestry-4.0.2.jar:/Users/tol/.m2/repository/net/sourceforge/htmlunit/htmlunit/2.19/htmlunit-2.19.jar:/Users/tol/.m2/repository/xalan/xalan/2.7.2/xalan-2.7.2.jar:/Users/tol/.m2/repository/xalan/serializer/2.7.2/serializer-2.7.2.jar:/Users/tol/.m2/repository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpclient/4.5.1/httpclient-4.5.1.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpcore/4.4.3/httpcore-4.4.3.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpmime/4.5.1/httpmime-4.5.1.jar:/Users/tol/.m2/repository/commons-codec/commons-codec/1.10/commons-codec-1.10.jar:/Users/tol/.m2/repository/net/sourceforge/htmlunit/htmlunit-core-js/2.17/htmlunit-core-js-2.17.jar:/Users/tol/.m2/repository/xerces/xercesImpl/2.11.0/xercesImpl-2.11.0.jar:/Users/tol/.m2/repository/xml-apis/xml-apis/1.4.01/xml-apis-1.4.01.jar:/Users/tol/.m2/repository/net/sourceforge/nekohtml/nekohtml/1.9.22/nekohtml-1.9.22.jar:/Users/tol/.m2/repository/net/sourceforge/cssparser/cssparser/0.9.18/cssparser-0.9.18.jar:/Users/tol/.m2/repository/commons-logging/commons-logging/1.2/commons-logging-1.2.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-client/9.2.13.v20150730/websocket-client-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-common/9.2.13.v20150730/websocket-common-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-api/9.2.13.v20150730/websocket-api-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-webapp/9.2.14.v20151106/jetty-webapp-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-xml/9.2.14.v20151106/jetty-xml-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-servlet/9.2.14.v20151106/jetty-servlet-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-security/9.2.14.v20151106/jetty-security-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-servlets/9.2.14.v20151106/jetty-servlets-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-continuation/9.2.14.v20151106/jetty-continuation-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-http/9.2.14.v20151106/jetty-http-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-util/9.2.14.v20151106/jetty-util-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-io/9.2.14.v20151106/jetty-io-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-annotations/9.2.14.v20151106/jetty-annotations-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-plus/9.2.14.v20151106/jetty-plus-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-jndi/9.2.14.v20151106/jetty-jndi-9.2.14.v20151106.jar:/Users/tol/.m2/repository/javax/annotation/javax.annotation-api/1.2/javax.annotation-api-1.2.jar:/Users/tol/.m2/repository/org/eclipse/jetty/apache-jsp/9.2.14.v20151106/apache-jsp-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-server/9.2.14.v20151106/jetty-server-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/toolchain/jetty-schemas/3.1.M0/jetty-schemas-3.1.M0.jar:/Users/tol/.m2/repository/javax/servlet/javax.servlet-api/3.1.0/javax.servlet-api-3.1.0.jar:/Users/tol/.m2/repository/org/mortbay/jasper/apache-jsp/8.0.9.M3/apache-jsp-8.0.9.M3.jar:/Users/tol/.m2/repository/org/mortbay/jasper/apache-el/8.0.9.M3/apache-el-8.0.9.M3.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-elemental/2.8.0/gwt-elemental-2.8.0.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-user/2.8.0/gwt-user-2.8.0.jar:/Users/tol/.m2/repository/com/google/jsinterop/jsinterop-annotations/1.0.1/jsinterop-annotations-1.0.1.jar:/Users/tol/.m2/repository/com/google/jsinterop/jsinterop-annotations/1.0.1/jsinterop-annotations-1.0.1-sources.jar:/Users/tol/.m2/repository/javax/validation/validation-api/1.0.0.GA/validation-api-1.0.0.GA.jar:/Users/tol/.m2/repository/javax/validation/validation-api/1.0.0.GA/validation-api-1.0.0.GA-sources.jar:/Users/tol/.m2/repository/org/w3c/css/sac/1.3/sac-1.3.jar:/Users/tol/.m2/repository/com/github/nmorel/gwtjackson/gwt-jackson/0.14.2/gwt-jackson-0.14.2.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.8.4/jackson-annotations-2.8.4.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.8.4/jackson-annotations-2.8.4-sources.jar:/Users/tol/.m2/repository/com/squareup/javapoet/1.0.0/javapoet-1.0.0.jar:/Users/tol/.m2/repository/junit/junit/4.12/junit-4.12.jar:/Users/tol/.m2/repository/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar:/Users/tol/.m2/repository/com/machinepublishers/jbrowserdriver/0.17.5/jbrowserdriver-0.17.5.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpclient-cache/4.5.2/httpclient-cache-4.5.2.jar:/Users/tol/.m2/repository/org/slf4j/slf4j-api/1.7.2/slf4j-api-1.7.2.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-api/2.53.0/selenium-api-2.53.0.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-remote-driver/2.53.0/selenium-remote-driver-2.53.0.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-server/2.53.0/selenium-server-2.53.0.jar:/Users/tol/.m2/repository/com/google/guava/guava/19.0/guava-19.0.jar:/Users/tol/.m2/repository/org/zeroturnaround/zt-process/1.3/zt-process-1.3.jar:/Users/tol/.m2/repository/org/zeroturnaround/zt-exec/1.7/zt-exec-1.7.jar:/Users/tol/.m2/repository/com/sun/jna/jna/3.0.9/jna-3.0.9.jar:/Users/tol/.m2/repository/commons-lang/commons-lang/2.6/commons-lang-2.6.jar:/Users/tol/.m2/repository/io/github/lukehutch/fast-classpath-scanner/2.0.16/fast-classpath-scanner-2.0.16.jar:/Users/tol/.m2/repository/io/vertx/vertx-core/3.4.1/vertx-core-3.4.1.jar:/Users/tol/.m2/repository/io/netty/netty-common/4.1.8.Final/netty-common-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-buffer/4.1.8.Final/netty-buffer-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-transport/4.1.8.Final/netty-transport-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-handler/4.1.8.Final/netty-handler-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec/4.1.8.Final/netty-codec-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-handler-proxy/4.1.8.Final/netty-handler-proxy-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-socks/4.1.8.Final/netty-codec-socks-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-http/4.1.8.Final/netty-codec-http-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-http2/4.1.8.Final/netty-codec-http2-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-resolver/4.1.8.Final/netty-resolver-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-resolver-dns/4.1.8.Final/netty-resolver-dns-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-dns/4.1.8.Final/netty-codec-dns-4.1.8.Final.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.7.4/jackson-core-2.7.4.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.7.4/jackson-databind-2.7.4.jar:/Applications/IntelliJ%20IDEA.app/Contents/lib/idea_rt.jar:/Volumes/HDD/work/vertxui/vertxui-examples/src/main/java"
Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI translate
INFO: Starting GWT with commandline: java -cp "/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/charsets.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/deploy.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/cldrdata.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/dnsns.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/jaccess.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/jfxrt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/localedata.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/nashorn.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/sunec.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/sunjce_provider.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/sunpkcs11.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/ext/zipfs.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/javaws.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/jce.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/jfr.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/jfxswt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/jsse.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/management-agent.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/plugin.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/resources.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre/lib/rt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/ant-javafx.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/dt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/javafx-mx.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/jconsole.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/packager.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/sa-jdi.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/lib/tools.jar:/Volumes/HDD/work/vertxui/vertxui-examples/target/classes/:/Volumes/HDD/work/vertxui/vertxui-core/target/classes/:/Users/tol/.m2/repository/io/vertx/vertx-web/3.4.1/vertx-web-3.4.1.jar:/Users/tol/.m2/repository/io/vertx/vertx-auth-common/3.4.1/vertx-auth-common-3.4.1.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-dev/2.8.0/gwt-dev-2.8.0.jar:/Users/tol/.m2/repository/com/google/code/gson/gson/2.6.2/gson-2.6.2.jar:/Users/tol/.m2/repository/org/ow2/asm/asm/5.0.3/asm-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-util/5.0.3/asm-util-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-tree/5.0.3/asm-tree-5.0.3.jar:/Users/tol/.m2/repository/org/ow2/asm/asm-commons/5.0.3/asm-commons-5.0.3.jar:/Users/tol/.m2/repository/colt/colt/1.2.0/colt-1.2.0.jar:/Users/tol/.m2/repository/ant/ant/1.6.5/ant-1.6.5.jar:/Users/tol/.m2/repository/commons-collections/commons-collections/3.2.2/commons-collections-3.2.2.jar:/Users/tol/.m2/repository/commons-io/commons-io/2.4/commons-io-2.4.jar:/Users/tol/.m2/repository/com/ibm/icu/icu4j/50.1.1/icu4j-50.1.1.jar:/Users/tol/.m2/repository/tapestry/tapestry/4.0.2/tapestry-4.0.2.jar:/Users/tol/.m2/repository/net/sourceforge/htmlunit/htmlunit/2.19/htmlunit-2.19.jar:/Users/tol/.m2/repository/xalan/xalan/2.7.2/xalan-2.7.2.jar:/Users/tol/.m2/repository/xalan/serializer/2.7.2/serializer-2.7.2.jar:/Users/tol/.m2/repository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpclient/4.5.1/httpclient-4.5.1.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpcore/4.4.3/httpcore-4.4.3.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpmime/4.5.1/httpmime-4.5.1.jar:/Users/tol/.m2/repository/commons-codec/commons-codec/1.10/commons-codec-1.10.jar:/Users/tol/.m2/repository/net/sourceforge/htmlunit/htmlunit-core-js/2.17/htmlunit-core-js-2.17.jar:/Users/tol/.m2/repository/xerces/xercesImpl/2.11.0/xercesImpl-2.11.0.jar:/Users/tol/.m2/repository/xml-apis/xml-apis/1.4.01/xml-apis-1.4.01.jar:/Users/tol/.m2/repository/net/sourceforge/nekohtml/nekohtml/1.9.22/nekohtml-1.9.22.jar:/Users/tol/.m2/repository/net/sourceforge/cssparser/cssparser/0.9.18/cssparser-0.9.18.jar:/Users/tol/.m2/repository/commons-logging/commons-logging/1.2/commons-logging-1.2.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-client/9.2.13.v20150730/websocket-client-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-common/9.2.13.v20150730/websocket-common-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/websocket/websocket-api/9.2.13.v20150730/websocket-api-9.2.13.v20150730.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-webapp/9.2.14.v20151106/jetty-webapp-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-xml/9.2.14.v20151106/jetty-xml-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-servlet/9.2.14.v20151106/jetty-servlet-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-security/9.2.14.v20151106/jetty-security-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-servlets/9.2.14.v20151106/jetty-servlets-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-continuation/9.2.14.v20151106/jetty-continuation-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-http/9.2.14.v20151106/jetty-http-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-util/9.2.14.v20151106/jetty-util-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-io/9.2.14.v20151106/jetty-io-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-annotations/9.2.14.v20151106/jetty-annotations-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-plus/9.2.14.v20151106/jetty-plus-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-jndi/9.2.14.v20151106/jetty-jndi-9.2.14.v20151106.jar:/Users/tol/.m2/repository/javax/annotation/javax.annotation-api/1.2/javax.annotation-api-1.2.jar:/Users/tol/.m2/repository/org/eclipse/jetty/apache-jsp/9.2.14.v20151106/apache-jsp-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/jetty-server/9.2.14.v20151106/jetty-server-9.2.14.v20151106.jar:/Users/tol/.m2/repository/org/eclipse/jetty/toolchain/jetty-schemas/3.1.M0/jetty-schemas-3.1.M0.jar:/Users/tol/.m2/repository/javax/servlet/javax.servlet-api/3.1.0/javax.servlet-api-3.1.0.jar:/Users/tol/.m2/repository/org/mortbay/jasper/apache-jsp/8.0.9.M3/apache-jsp-8.0.9.M3.jar:/Users/tol/.m2/repository/org/mortbay/jasper/apache-el/8.0.9.M3/apache-el-8.0.9.M3.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-elemental/2.8.0/gwt-elemental-2.8.0.jar:/Users/tol/.m2/repository/com/google/gwt/gwt-user/2.8.0/gwt-user-2.8.0.jar:/Users/tol/.m2/repository/com/google/jsinterop/jsinterop-annotations/1.0.1/jsinterop-annotations-1.0.1.jar:/Users/tol/.m2/repository/com/google/jsinterop/jsinterop-annotations/1.0.1/jsinterop-annotations-1.0.1-sources.jar:/Users/tol/.m2/repository/javax/validation/validation-api/1.0.0.GA/validation-api-1.0.0.GA.jar:/Users/tol/.m2/repository/javax/validation/validation-api/1.0.0.GA/validation-api-1.0.0.GA-sources.jar:/Users/tol/.m2/repository/org/w3c/css/sac/1.3/sac-1.3.jar:/Users/tol/.m2/repository/com/github/nmorel/gwtjackson/gwt-jackson/0.14.2/gwt-jackson-0.14.2.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.8.4/jackson-annotations-2.8.4.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.8.4/jackson-annotations-2.8.4-sources.jar:/Users/tol/.m2/repository/com/squareup/javapoet/1.0.0/javapoet-1.0.0.jar:/Users/tol/.m2/repository/junit/junit/4.12/junit-4.12.jar:/Users/tol/.m2/repository/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar:/Users/tol/.m2/repository/com/machinepublishers/jbrowserdriver/0.17.5/jbrowserdriver-0.17.5.jar:/Users/tol/.m2/repository/org/apache/httpcomponents/httpclient-cache/4.5.2/httpclient-cache-4.5.2.jar:/Users/tol/.m2/repository/org/slf4j/slf4j-api/1.7.2/slf4j-api-1.7.2.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-api/2.53.0/selenium-api-2.53.0.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-remote-driver/2.53.0/selenium-remote-driver-2.53.0.jar:/Users/tol/.m2/repository/org/seleniumhq/selenium/selenium-server/2.53.0/selenium-server-2.53.0.jar:/Users/tol/.m2/repository/com/google/guava/guava/19.0/guava-19.0.jar:/Users/tol/.m2/repository/org/zeroturnaround/zt-process/1.3/zt-process-1.3.jar:/Users/tol/.m2/repository/org/zeroturnaround/zt-exec/1.7/zt-exec-1.7.jar:/Users/tol/.m2/repository/com/sun/jna/jna/3.0.9/jna-3.0.9.jar:/Users/tol/.m2/repository/commons-lang/commons-lang/2.6/commons-lang-2.6.jar:/Users/tol/.m2/repository/io/github/lukehutch/fast-classpath-scanner/2.0.16/fast-classpath-scanner-2.0.16.jar:/Users/tol/.m2/repository/io/vertx/vertx-core/3.4.1/vertx-core-3.4.1.jar:/Users/tol/.m2/repository/io/netty/netty-common/4.1.8.Final/netty-common-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-buffer/4.1.8.Final/netty-buffer-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-transport/4.1.8.Final/netty-transport-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-handler/4.1.8.Final/netty-handler-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec/4.1.8.Final/netty-codec-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-handler-proxy/4.1.8.Final/netty-handler-proxy-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-socks/4.1.8.Final/netty-codec-socks-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-http/4.1.8.Final/netty-codec-http-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-http2/4.1.8.Final/netty-codec-http2-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-resolver/4.1.8.Final/netty-resolver-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-resolver-dns/4.1.8.Final/netty-resolver-dns-4.1.8.Final.jar:/Users/tol/.m2/repository/io/netty/netty-codec-dns/4.1.8.Final/netty-codec-dns-4.1.8.Final.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.7.4/jackson-core-2.7.4.jar:/Users/tol/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.7.4/jackson-databind-2.7.4.jar:/Applications/IntelliJ%20IDEA.app/Contents/lib/idea_rt.jar:/Volumes/HDD/work/vertxui/vertxui-examples/src/main/java" com.google.gwt.dev.Compiler -strict -XdisableUpdateCheck -war /Volumes/HDD/work/vertxui/vertxui-examples/build/development -draftCompile -optimize 0 -style DETAILED gwtTemp
Dec 02, 2017 4:45:35 PM live.connector.vertxui.samples.server.AllExamplesServer lambda$start$1
INFO: Initialised:
	http://localhost:8088/rest/totals
	http://localhost:8088/rest/bills
	http://localhost:8088/rest/grocery
	http://localhost:8088/figwheely.js
	http://localhost:8088/
.Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI translateContinue
INFO: Gwt says: Loading inherited module 'gwtTemp'
Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI translateContinue
INFO: Gwt says:    [ERROR] Unable to find 'gwtTemp.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?
.Dec 02, 2017 4:45:35 PM live.connector.vertxui.server.VertxUI writeHtml
INFO: Writing final index.html to: build/development/index.html
Compile error(s): Loading inherited module 'gwtTemp'
   [ERROR] Unable to find 'gwtTemp.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?

```

--- a/vertxui-core/src/main/java/live/connector/vertxui/server/VertxUI.java
+++ b/vertxui-core/src/main/java/live/connector/vertxui/server/VertxUI.java
@@ -253,7 +253,7 @@ public class VertxUI {
 				? (System.getProperty("java.class.path").contains(";") ? ";" : ":")
 				: System.getenv("path.separator");
 		String classpath = System.getProperty("java.class.path");
-		classpath = "\"" + classpath + separator + new File(folderSource).getAbsolutePath() + "\"";
+		classpath = '"' + separator + classpath + separator + new File(folderSource).getAbsolutePath()  + separator + '"';
 		log.fine("Classpath = " + classpath);
 
 		// Check whether the classpath contains gwt

